scout/server/blueprints/variants/controllers.py

```diff
@@ -374,9 +374,11 @@ def variant_export_lines(store, case_obj, variants_query):
             variant_line += gene_info
         else:
             i = 0
-            while i < 4:
-                variant_line.append("-")  # instead of gene ids
-                i = i + 1
+            while i < 3:
```

Comment:
Unnamed constant in while-loop (4).

Reply:
It's a good suggestion, I'll rename it to empty_col, thanks!